lib/command.js

```diff
@@ -16,6 +16,13 @@ Command.prototype.add = function add (name, fn) {
   this._cmds.set(name, fn);
 };
 
+Command.prototype.list = function list () {
+  const commands = [];
```

Comment:
If you use `Object.keys()`, then you can do `const commands = new Array(size);`, which is faster. Then in your loop, you would assign to specific indexes instead of using `push()`.